po/fr.po

```diff
@@ -131,319 +144,452 @@ msgid "Highlight focused application"
+msgstr "1 fenêtre ouverte (ou dégroupée)"
+
+#: Settings.ui.h:23
 msgid "Apply to all"
 msgstr "Appliquer à tout"
 
-#: Settings.ui.h:21
+#: Settings.ui.h:24
 msgid "2 windows open"
 msgstr "2 fenêtres ouvertes"
 
-#: Settings.ui.h:22
+#: Settings.ui.h:25
 msgid "3 windows open"
 msgstr "3 fenêtres ouvertes"
 
-#: Settings.ui.h:23
+#: Settings.ui.h:26
 msgid "4+ windows open"
 msgstr "4+ fenêtres ouvertes"
 
-#: Settings.ui.h:24
+#: Settings.ui.h:27
 msgid "Use different for unfocused"
 msgstr "Style différent pour les applications inactives"
 
-#: Settings.ui.h:25
+#: Settings.ui.h:28
+msgid "Font size (px) of the application titles (default is 14)"
+msgstr "Taille (px) du texte des titres d'application (défaut: 14)"
+
+#: Settings.ui.h:29
+msgid "Font color of the application titles"
+msgstr "Couleur du texte des titres d'application"
+
+#: Settings.ui.h:30
+msgid "Maximum width (px) of the application titles (default is 160)"
+msgstr "Longueur maximum (px) des titres d'application (défaut: 160)"
+
+#: Settings.ui.h:31
+#, fuzzy
+msgid "Use a fixed width for the application titles"
+msgstr "Utiliser une largeur fixe pour les titres d'application"
+
+#: Settings.ui.h:32
+msgid ""
+"The application titles all have the same width, even if their texts are "
+"shorter than the maximum width. The maximum width value is used as the fixed "
+"width."
+msgstr ""
+"Les titres d'application ont tous la même longueur, même si leur texte est "
+"plus petit que la taille maximum. La valeur maximale de longueur est "
+"utilisée comme longueur fixe."
+
+#: Settings.ui.h:33
+msgid "Display running indicators on unfocused applications"
+msgstr "Afficher des indicateurs sur les applications an arrière-plan"
+
+#: Settings.ui.h:34
+msgid "Use the favorite icons as application launchers"
+msgstr "Utiliser les applications favorites comme lanceurs"
+
+#: Settings.ui.h:35
 msgid "Preview timeout on icon leave (ms)"
 msgstr "Délai de l'aperçu lorsqu'on quitte l'icône (ms)"
 
-#: Settings.ui.h:26
+#: Settings.ui.h:36
 msgid ""
 "If set too low, the window preview of running applications may seem to close "
 "too quickly when trying to enter the popup. If set too high, the preview may "
 "linger too long when moving to an adjacent icon."
 msgstr ""
-"Si trop bas, l'aperçu des fenêtres des applications actives peut "
-"sembler se fermer trop vite quand on essaie de cliquer sur l'aperçu. Si "
-"trop haut, l'aperçu peut rester trop longtemps quand on passe à une icône "
-"adjacente."
+"Si trop bas, l'aperçu des fenêtres des applications actives peut sembler se "
+"fermer trop vite quand on essaie de cliquer sur l'aperçu. Si trop haut, "
+"l'aperçu peut rester trop longtemps quand on passe à une icône adjacente."
 
-#: Settings.ui.h:27
+#: Settings.ui.h:37
+msgid "Time (ms) before showing (100 is default)"
+msgstr "Temps (ms) avant d'afficher (100 par défaut)"
+
+#: Settings.ui.h:38
+msgid "Enable window peeking"
+msgstr "Activer la vue de fenêtre"
```

Comment:
Sounds weird, I'd use something like "Activer la prévisualisation" or "miniature"

Reply:
"Window peeking" is definitely weird to translate though. I did not remember how it was called in Windows (given the obvious inspiration). "prévisualisation" seems a good word, and I have used it elsewhere too.